includes/wikia/services/RevisionUpvotesService.class.php

```diff
@@ -42,7 +42,7 @@ public function removeUpvote( $id, $fromUser, $userId ) {
 		$db->begin();
 
 		( new \WikiaSQL() )
-			->DELETE( self::UPVOTE_TABLE )
+			->DELETE( static::UPVOTE_TABLE )
 			->WHERE( 'id' )->EQUAL_TO( $id )
 			->AND_( 'from_user' )->EQUAL_TO( $fromUser )
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Define a constant instead of duplicating this literal "from_user" 5 times. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://10.8.74.32:9000/coding_rules#rule_key=php%3AS1192)